pkg/cri/resource-manager/policy/builtin/topology-aware/topology-aware-policy.go

```diff
@@ -164,7 +164,27 @@ func (p *policy) UpdateResources(c cache.Container) error {
 
 // ExportResourceData provides resource data to export for the container.
 func (p *policy) ExportResourceData(c cache.Container, syntax policyapi.DataSyntax) []byte {
-	return nil
+	grant, ok := p.allocations.CPU[c.GetCacheID()]
+	if !ok {
+		return []byte{}
+	}
+
+	shared := grant.SharedCPUs().String()
+	isolated := grant.ExclusiveCPUs().Intersection(grant.GetNode().GetCPU().IsolatedCPUs())
+	exclusive := grant.ExclusiveCPUs().Difference(isolated).String()
+
+	data := ""
+	if shared != "" {
+		data += "SHARED_CPUS=\"" + shared + "\"\n"
+	}
+	if isolated.String() != "" {
+		data += "ISOLATED_CPUS=\"" + isolated.String() + "\"\n"
```

Comment:
why do string conversion here, and not above like with others?

Reply:
We need isolated in CPUSet form so we can .Intersect() it to calculate 'exclusive'. The other fully regular way would have been to not .String() any of them, then check for emptiness using (cpuset.CPUSet).IsEmtpy() and finally .String()ify them during string concatenation. Dunno which is better... I can change it to be like that if it is considered preferable/better.